src/main/java/graphql/annotations/DefaultTypeFunction.java

```diff
@@ -355,6 +353,18 @@ public DefaultTypeFunction() {
         typeFunctions.add(new ObjectFunction());
     }
 
+    @Reference(cardinality = ReferenceCardinality.MULTIPLE,
```

Comment:
Note: with `cardinality = ReferenceCardinality.MULTIPLE` we can use default policy option `ReferencePolicyOption.RELUCTANT`, therefore `ReferencePolicyOption.GREEDY` as was used before is not needed.